astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -1844,7 +1844,7 @@ def test_single_line_string(delimiter, fast_reader):
     at least treated consistently.
     """
     expected = Table([[1], [2], [3.00]], names=("col1", "col2", "col3"))
-    text = "1{0:s}2{0:s}3.0".format(delimiter)
+    text = f"1{delimiter:s}2{delimiter:s}3.0"
```

Comment:
Agreed. I find it annoying that ruff adds that.

Reply:
I find this one of the cases where `.format()` has its justification, allowing to reuse its argument. Arguably this is borderline beneficial here, but I would not set a precedent to disable it for more repeated usage, or e.g. the
`"A: {key1} B: {key2} C: {key3}".format(dict_of_args)` style.
It would be OK to change here though, just bringing it up for discussion.